sklearn/preprocessing/_data.py

```diff
@@ -2715,6 +2715,8 @@ class PowerTransformer(TransformerMixin, BaseEstimator):
 
     .. [2] G.E.P. Box and D.R. Cox, "An Analysis of Transformations", Journal
            of the Royal Statistical Society B, 26, 211-252 (1964).
+
+    .. versionadded:: 0.19.2
```

Comment:
Is there a reason you specified 0.19.2 as opposed to 0.19 ?

Reply:
I double checked, and it seems to have been added in 0.20, as stated in the `doc/whats_new/v0.20.rst` file. Where did you find the 0.19.2 mention ?